client/src/Components/Categories/Categories.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./category.scss";
+
+const Categories = ({ categoryList }) => {
+
+  const orderedCategories = categoryList
+    .filter((item) => item.order != -1)
+    .sort((item1, item2) => {
+      return item1.order - item2.order;
+    });
```

Comment:
extract the filter sort logic to the parent component, or any other place like state. Categories component shouldvrender items as is